### ISSUE TITLE
[BUG] Various bugs

### ISSUE THREAD

**Issue by the1gofer** [mdelobelle/metadatamenu#492]
I've run into a couple of potential bus

**1. When I was using the button in table view to add a link, it was rendered as text.** 

<img width="739" alt="Screenshot 2023-12-24 at 10 50 48 AM" src="https://github.com/mdelobelle/metadatamenu/assets/2140724/9f41c47d-4cec-4155-b681-852a44569b79">

**2. Also, clicking any of the links, doesn't do anything.** 

**3. metadata buttons are rendered when using a template to fill the note.**
<img width="337" alt="Screenshot 2023-12-24 at 11 49 29 AM" src="https://github.com/mdelobelle/metadatamenu/assets/2140724/4414394b-46ea-4aef-9f23-21b857935435">

Note source:
```
---
fileClass: Investigate Case
defendant(s):  
attorney(s): 
call count: 0
---
```

template source:
```
---
fileClass: Investigate Case
defendant(s):  
attorney(s): 
call count: 0
---
```

**Comment by mdelobelle** [comment]
> 2. Also, clicking any of the links, doesn't do anything.

will be fixed in the next release

**Comment by mdelobelle** [comment]
What is the type of "defendant(s)" currently ?
Multi Select or Multi File?

**Comment by mdelobelle** [comment]
should be fixed in 0.7.7
